src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public enum Colors {

```

Comment:
```suggestion
public enum Color {
```
we call enum singular